### PR TITLE
Adds varnish 4.1.3 since varnish 5.0.0 is now latest stable in homebrew/core

### PR DESCRIPTION
**Pull request by quintonparker**

### CODE REVIEW

File: varnish4.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  homepage "https://www.varnish-cache.org/"
+  url "https://repo.varnish-cache.org/source/varnish-4.1.3.tar.gz"
+  sha256 "9f9469b9fda2a578da2a9d282c71c34eeb5c42eda7f8d8728284d92282108429"
+
+  bottle do
+    rebuild 1
+    sha256 "ad41b596be38cc6fd5dcaef1b8108ad6ebb8cdded562f0fa59c81daa9cff0c93" => :el_capitan
+    sha256 "1c5b01ad96283950662fecb65254274a6c8481f4d130e1f1dbac1af821abec4a" => :yosemite
+    sha256 "270560b2d7bf43f660c9b126b68ce8df6f2226f93a472615d1c29e186d84da0a" => :mavericks
+  end
```
Comment (MikeMcQuaid) [inline varnish4.rb]: Remove the `bottle do` block.

File: varnish4.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--localstatedir=#{var}"
+    system "make", "install"
+    (etc+"varnish").install "etc/example.vcl" => "default.vcl"
+    (var+"varnish").mkpath
+  end
+
+  plist_options manual: "#{HOMEBREW_PREFIX}/sbin/varnishd -n #{HOMEBREW_PREFIX}/var/varnish -f #{HOMEBREW_PREFIX}/etc/varnish/default.vcl -s malloc,1G -T 127.0.0.1:2000 -a 0.0.0.0:8080"
```
Comment (MikeMcQuaid) [inline varnish4.rb]: Replace `manual:` with `:manual =>`

File: varnish4.rb (hunk 1 above)
Comment (quintonparker) [inline varnish4.rb]: done

File: varnish4.rb (hunk 2 above)
Comment (quintonparker) [inline varnish4.rb]: done

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by quintonparker** [comment]
no. thank you chaps for homebrew! appreciate the warm reception

**Pull request merged**